docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md

```diff
@@ -0,0 +1,39 @@
+# Existing EKS Cluster AWS Mixed Observability Accelerator
+
+## Objective
+
+This pattern depends on the previous deployment of [_Single New EKS Cluster Observability Accelerator_](./single-new-eks-cluster.md).
+After deploying this pattern you will have the same setup as in [_Single New EKS Cluster AWS Mixed Observability Accelerator_](./single-new-eks-mixed-observability.md).
+
```

Comment:
pre req i mentioned i smissing
1. EKS Cluster pointing to single eks cluster as example.
2. OIDC provider setup for EKS cluster. (Single EKS Cluster takes care of it).